builtin/providers/circonus/resource_circonus_check_cloudwatch_test.go

```diff
@@ -23,192 +23,192 @@ func TestAccCirconusCheckCloudWatch_basic(t *testing.T) {
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.dimmensions.DBInstanceIdentifier", "atlas-production"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.#", "17"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.990896688", "CPUUtilization"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.3895259375", "DatabaseConnections"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.1328149445", "DiskQueueDepth"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.4218650584", "FreeStorageSpace"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.1835248983", "FreeableMemory"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.2757008135", "MaximumUsedTransactionIDs"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.915415866", "NetworkReceiveThroughput"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.1852047735", "NetworkTransmitThroughput"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.3518416306", "ReadIOPS"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.114013313", "ReadLatency"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.1284099341", "ReadThroughput"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.4205329773", "SwapUsage"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.3550163941", "TransactionLogsDiskUsage"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.2231806695", "TransactionLogsGeneration"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.335777904", "WriteIOPS"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.3894876280", "WriteLatency"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.metric.1569904650", "WriteThroughput"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.namespace", "AWS/RDS"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.version", "2010-08-01"),
-					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.1637274235.url", "https://monitoring.us-east-1.amazonaws.com"),
+					resource.TestCheckResourceAttr("circonus_check.rds_metrics", "cloudwatch.88938937.dimmensions.%", "1"),
```

Comment:
Correct.  I'm going to make a sweep for #12808 soon, but yeah, this PR isn't addressing that.  This PR is just chasing some tag drift that I noticed from the tests and chasing the checksum drift as a result.

re: #12808 I didn't realize until you brought it up that there was a special case for `TypeList` with a `MaxItems=1`, but yeah, I'm definitely going to make that conversion.  Chasing these checksums is tedious.  ;)